web_ui/frontend/app/api/docs/pelican-swagger.yaml

```diff
@@ -329,6 +333,31 @@ definitions:
         type: number
         description: The longitude of the server based on its IP address
         default: 0
+      enableWrite:
```

Comment:
These too

Reply:
If we changed the json names, we will have backward compatibility issue as the data was no longer the same. Do you think we should do a separate API or keep json name untouched for now?

Reply:
Ah, I didn't look closely enough and didn't realize these were going to affect the underlying JSON.